interpreter/cling/include/cling/libc.modulemap

```diff
@@ -28,14 +28,16 @@ module "libc" [system] [extern_c] [no_undeclared_includes] {
     export *
     header "math.h"
   }
-/*  module "pthread.h" {
+  module "pthread.h" {
+	requires !header_existence
     export *
     header "pthread.h"
-  }*/
-/*  module "setjmp.h" {
+  }
+  module "setjmp.h" {
+	requires !header_existence
```

Comment:
Also, before each line please add a comment saying that is required by windows.

Reply:
Added a commit for it.